### ISSUE TITLE
[Bug]: Readme documents the ability to scrape a web page that seems absent from the actual binary

### ISSUE THREAD

**Issue by DarwinJS** [danielmiessler/fabric#1138]
### What happened?

v1.4.100

The readme here: https://github.com/danielmiessler/fabric?tab=readme-ov-file#usage

Says 
"  -U, --updatepatterns       Update patterns"
"  -u, --scrape_url=          Scrape website URL to markdown using Jina AI"

Fabric -h says:
"  --update, -u          Update patterns"

My actual experience of -u on v1.4.100 does a pattern update.

Trying to use --scrape_url gives:
`fabric: error: unrecognized arguments: --scrape_url=<the url used>`

So I can't find out if and how fabric can read a webpage as input?

### Version check

- [X] Yes I was.

**Comment by DarwinJS** [comment]
Is curling it and piping an effective method or does the raw html get in the way?

**Comment by ksylvan** [comment]
@DarwinJS you can use `pandoc` to turn html to plaintext, so curl some web page and pipe to pandoc and to fabric

**Comment by johnsaigle** [comment]
`fabric-ai -u` works for me as expected. Version: `v1.4.185` installed via homebrew.

**Comment by ksylvan** [comment]
Yeah, you are correct. `fabric -U` updates patterns for me and `fabric -u` does the web scraping.

I learned something new about fabric!

```plaintext
$ fabric -u 'https://github.com/danielmiessler/fabric/issues/1138' -p summarize_micro 
### ONE SENTENCE SUMMARY:
Fabric's README mentions web scraping feature absent in binary v1.4.100.

### MAIN POINTS:
- README claims Fabric can scrape URLs using Jina AI.
- Binary v1.4.100 lacks --scrape_url functionality, shows error.
- Later version v1.4.185 reportedly supports URL scraping.

### TAKEAWAYS:
- Check Fabric version for URL scraping compatibility.
- Use pandoc with curl as workaround for scraping.
- Update to latest version for potential fixes.
```
